### PR TITLE
starting a re-org of the dataproc dir to accommodate new content + phasing out of older content

### PR DESCRIPTION
**Pull request by amygdala**

### CODE REVIEW

File: dataproc/README.md [hunk 1]
```diff
@@ -1,7 +1,24 @@
...
+```sh
+git clone https://github.com/DataBiosphere/terra-axon-examples.git
+```
+You can also upload the notebook files directly to the JupyterLab server.
+
+The [annotate_significant_gwas_results_with_gnomad.ipynb](./annotate_significant_gwas_results_with_gnomad.ipynb) notebook uses [Hail](https://hail.is/) to annotate the significant GWAS results with [gnomAD](https://gnomad.broadinstitute.org/).
+This notebook is intended to be run from the on-cluster JupyterLab server, and requires that Hail is installed on cluster creation.
+
+The [batch_job_submit.ipynb](./batch_job_submit.ipynb) notebook shows an example of submitting a batch job to the Dataproc cluster.  It uses the  [annotate_significant_gwas_results_with_gnomad.ipynb](./annotate_significant_gwas_results_with_gnomad.ipynb) notebook file as a basis for its example batch script, but other scripts can be submitted in the same manner.
+This notebook requires a running Dataproc cluster, but can itself be run from either the on-cluster JupyerLab server, or from a  [notebook cloud environment](https://support.workbench.verily.com/docs/how_to_guides/using_cloud_environments/).
```
Comment (wnojopra) [inline dataproc/README.md]: nit
```suggestion
This notebook requires a running Dataproc cluster, but can itself be run from either the on-cluster JupyerLab server, or from a [notebook cloud environment](https://support.workbench.verily.com/docs/how_to_guides/using_cloud_environments/).
```

File: dataproc/README.md [hunk 2]
```diff
@@ -1,7 +1,24 @@
...
+Otherwise, you can directly clone the repo in the notebook server yourself (e.g., from the Terminal window), via:
+```sh
+git clone https://github.com/DataBiosphere/terra-axon-examples.git
+```
+You can also upload the notebook files directly to the JupyterLab server.
+
+The [annotate_significant_gwas_results_with_gnomad.ipynb](./annotate_significant_gwas_results_with_gnomad.ipynb) notebook uses [Hail](https://hail.is/) to annotate the significant GWAS results with [gnomAD](https://gnomad.broadinstitute.org/).
+This notebook is intended to be run from the on-cluster JupyterLab server, and requires that Hail is installed on cluster creation.
+
+The [batch_job_submit.ipynb](./batch_job_submit.ipynb) notebook shows an example of submitting a batch job to the Dataproc cluster.  It uses the  [annotate_significant_gwas_results_with_gnomad.ipynb](./annotate_significant_gwas_results_with_gnomad.ipynb) notebook file as a basis for its example batch script, but other scripts can be submitted in the same manner.
```
Comment (wnojopra) [inline dataproc/README.md]: nit
```suggestion
The [batch_job_submit.ipynb](./batch_job_submit.ipynb) notebook shows an example of submitting a batch job to the Dataproc cluster. It uses the [annotate_significant_gwas_results_with_gnomad.ipynb](./annotate_significant_gwas_results_with_gnomad.ipynb) notebook file as a basis for its example batch script, but other scripts can be submitted in the same manner.
```

File: dataproc/annotate_significant_gwas_results_with_gnomad.ipynb [hunk 3]
```diff
@@ -64,7 +64,42 @@
...
+    "This staging bucket is automatically created when you first spin up a Dataproc cluster.  If you want to use a different bucket (which must already exist) you can edit the code below."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "STAGING_BUCKET_CMD_OUTPUT = !terra resolve --name=dataproc-staging-{WORKSPACE_ID}\n",
```
Comment (wnojopra) [inline dataproc/annotate_significant_gwas_results_with_gnomad.ipynb:99]: Does WORKSPACE_ID ever get truncated or renamed or anything? I wonder if grepping the output of terra resource list would be better. Your call on which is better. If you do end up changing, also change the similar code in batch_job_submit.

File: dataproc/dataproc_workspace_dashboard.md [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+## Step 0: Set up your github SSH key (optional) and _Duplicate_ this workspace
+
+- (Optional) In the Enterprise Terra UI, click on your profile icon, then click on "My Profile". Copy the public SSH key listed there.
+  Then, [add that key](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account) to your GitHub account. This is not necessary to run the Dataproc example notebooks, but will allow you to clone private repos from your workspace cloud environments.
+  [This page](https://terra-docs.api.verily.com/docs/how_to_guides/terra_ssh_key_guide/) has more detail on the process.
+
+- Then, *Duplicate* this workspace. You can do that via the 'three-dot' menu in the upper right of the workspace.
+
+
+<!-- ## Step 1: Preview prior runs of the relevant notebooks.
```
Comment (wnojopra) [inline dataproc/dataproc_workspace_dashboard.md:20]: Is this section commented out purposefully?

File: dataproc/annotate_significant_gwas_results_with_gnomad.ipynb [hunk 5]
```diff
@@ -86,14 +121,15 @@
     "# GRCh38 GWAS results from https://www.biorxiv.org/content/10.1101/2021.10.11.463514v1.supplementary-material\n",
     "SELVARAJ_LIPIDS_GWAS_RESULTS = 'https://www.biorxiv.org/content/biorxiv/early/2021/10/12/2021.10.11.463514/DC1/embed/media-1.xlsx?download=true'\n",
     "\n",
-    "# To run at scale, use these intervals for all autosomes.\n",
-    "#INTERVALS_TO_EXAMINE = ['chr1-chr22'] \n",
+    "\n",
     "# To perform a quick test, use these intervals for APOE\n",
     "# https://genome.ucsc.edu/cgi-bin/hgTracks?db=hg38&lastVirtModeType=default&lastVirtModeExtraState=&virtModeType=default&virtMode=0&nonVirtPosition=&position=chr19%3A44905796%2D44909393&hgsid=1411550209_Ip7W0XTmGpvEfKONETLScNT3MOVv\n",
     "INTERVALS_TO_EXAMINE = ['chr19:44905796-44909393']\n",
+    "# To run at scale, use these intervals for all autosomes.\n",
```
Comment (wnojopra) [inline dataproc/annotate_significant_gwas_results_with_gnomad.ipynb:128]: Would be good to comment here on about how long all autosomes would take.

**Review by wnojopra — APPROVED** [review]

File: dataproc/dataproc_workspace_dashboard.md (hunk 4 above)
Comment (amygdala) [reply dataproc/dataproc_workspace_dashboard.md:20]: Yeah, I need to generate new snapshots first..

File: dataproc/annotate_significant_gwas_results_with_gnomad.ipynb (hunk 3 above)
Comment (amygdala) [reply dataproc/annotate_significant_gwas_results_with_gnomad.ipynb:99]: good question... I'll check on that.

File: dataproc/annotate_significant_gwas_results_with_gnomad.ipynb (hunk 5 above)
Comment (amygdala) [reply dataproc/annotate_significant_gwas_results_with_gnomad.ipynb:128]: good idea, I'll make a note for the next iteration.

**Comment by amygdala** [comment]
thank you, @wnojopra !!

**Pull request merged**